### PR TITLE
Add ptp tests from baremetal deploy repo

### PR DESCRIPTION
**Pull request by fedepaol**
This PR moves the ptp tests we have under baremetal deploy under the current test suite.

Currently the tests just check that the operator is deployer properly, no functional test is in place yet.

### CODE REVIEW

**Comment by davidvossel** [comment]
```Running gofmt
gofmt -s -l `find . -path ./vendor -prune -o -type f -name '*.go' -print`
./functests/test_suite_test.go:13:1: expected 'STRING', found '<<'
./functests/test_suite_test.go:29:1: expected 'STRING', found 'var'
./functests/test_suite_test.go:39:2: expected 'STRING', found 'if'
./functests/test_suite_test.go:45:1: expected 'STRING', found 'var'
./functests/test_suite_test.go:56:1: expected 'STRING', found 'var'
./functests/test_suite_test.go:60:4: expected ')', found 'EOF'```

**Comment by fedepaol** [comment]
> ```
> gofmt -s -l `find . -path ./vendor -prune -o -type f -name '*.go' -print`
> ./functests/test_suite_test.go:13:1: expected 'STRING', found '<<'
> ./functests/test_suite_test.go:29:1: expected 'STRING', found 'var'
> ./functests/test_suite_test.go:39:2: expected 'STRING', found 'if'
> ./functests/test_suite_test.go:45:1: expected 'STRING', found 'var'
> ./functests/test_suite_test.go:56:1: expected 'STRING', found 'var'
> ./functests/test_suite_test.go:60:4: expected ')', found 'EOF'```
> ```

:open_mouth:

File: functests/test_suite_test.go [hunk 1]
```diff
@@ -10,6 +10,11 @@ import (
 	. "github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/reporters"
 	. "github.com/onsi/gomega"
+<<<<<<< Updated upstream
```
Comment (slintes) [inline functests/test_suite_test.go]: @fedepaol I guess this is the place gofmt is not happy with ;)

Comment (fedepaol) [reply functests/test_suite_test.go]: Yes I saw it, thanks.
Wonder how did I miss it

**Pull request merged**